hats/details.md

```diff
@@ -45,7 +65,7 @@ What is NOT shared between Hats
 
 What is shared between Hats
 ---------------------------
- * **Firefox Core**, which is used to run all Hats (so, Firefox Developer Edition, as it currently exists, is *not* a Hat of Firefox Stable);
+ * **Firefox Core**, which is used to run all Hats (so, Firefox Developer Edition is a Hat that can apply to  Firefox Release, Firefox Beta, Firefox Aurora or Firefox Nightly);
```

Comment:
Are we proposing Firefox Developer Edition become a Hat?  Because as it is it's a specific release (Aurora) plus Hat-like settings for developers.